src/main/resources/processes/PSU/PSU.bpmn

```diff
@@ -1,32 +1,36 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<bpmn:definitions xmlns:bpmn="http://www.omg.org/spec/BPMN/20100524/MODEL" xmlns:bpmndi="http://www.omg.org/spec/BPMN/20100524/DI" xmlns:dc="http://www.omg.org/spec/DD/20100524/DC" xmlns:camunda="http://camunda.org/schema/1.0/bpmn" xmlns:di="http://www.omg.org/spec/DD/20100524/DI" xmlns:modeler="http://camunda.org/schema/modeler/1.0" id="Definitions_1nlo3cm" targetNamespace="http://bpmn.io/schema/bpmn" exporter="Camunda Modeler" exporterVersion="4.11.0" modeler:executionPlatform="Camunda Platform" modeler:executionPlatformVersion="7.15.0">
+<bpmn:definitions xmlns:bpmn="http://www.omg.org/spec/BPMN/20100524/MODEL" xmlns:bpmndi="http://www.omg.org/spec/BPMN/20100524/DI" xmlns:dc="http://www.omg.org/spec/DD/20100524/DC" xmlns:camunda="http://camunda.org/schema/1.0/bpmn" xmlns:di="http://www.omg.org/spec/DD/20100524/DI" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:modeler="http://camunda.org/schema/modeler/1.0" id="Definitions_1nlo3cm" targetNamespace="http://bpmn.io/schema/bpmn" exporter="Camunda Modeler" exporterVersion="4.9.0" modeler:executionPlatform="Camunda Platform" modeler:executionPlatformVersion="7.15.0">
   <bpmn:process id="PSU" isExecutable="true">
     <bpmn:startEvent id="StartEvent_PSU">
       <bpmn:outgoing>Flow_0l0h7fj</bpmn:outgoing>
     </bpmn:startEvent>
     <bpmn:endEvent id="EndEvent_PSU">
       <bpmn:incoming>Flow_1sqnq8i</bpmn:incoming>
+      <bpmn:incoming>Flow_0v13kf6</bpmn:incoming>
     </bpmn:endEvent>
-    <bpmn:sequenceFlow id="Flow_0l0h7fj" sourceRef="StartEvent_PSU" targetRef="CallActivity_PSU_REGISTRATION" />
     <bpmn:callActivity id="CallActivity_PSU_TRIAGE" name="Triage" calledElement="STAGE">
       <bpmn:extensionElements>
         <camunda:in businessKey="#{execution.processBusinessKey}" />
         <camunda:in sourceExpression="${STAGE_TRIAGE}" target="StageType" />
         <camunda:in sourceExpression="${STAGE_TRIAGE}" target="StageWorkFlow" />
         <camunda:in variables="all" />
+        <camunda:in sourceExpression="${execution.processBusinessKey}" target="CaseUUID" />
```

Comment:
Does the CaseUUID still need passing down? I don't think it's still being used now the close case action is removed

Reply:
I don' think I will be able to remove this as caseUUID come up as null without this camunuda changes. So I have to include this changes.